packages/lesswrong/server/scripts/debuggingScripts.js

```diff
@@ -56,3 +56,147 @@ Vulcan.populateNotifications = async ({username,
     _.times(replyNotifications, () => createDummyComment(randomUser, {postId: post._id, parentCommentId: comment._id}));
   }
 }
+
+let loremIpsumTokens = [ "lorem", "ipsum", "dolor", "sit", "amet",
+  "consectetur", "adipiscing", "elit", "sed", "do", "eiusmod", "tempor",
+  "incididunt", "ut", "labore", "et" ];
+
+function getLoremIpsumToken() {
+  let index = Math.floor(Math.random() * loremIpsumTokens.length);
+  return loremIpsumTokens[index];
+}
+
+// Generate a given number of characters of lorem ipsum.
+function makeLoremIpsum(length) {
+  let result = [];
+  let lengthSoFar = 0;
+  
+  result.push("Lorem ");
+  lengthSoFar = "Lorem ".length;
+  
+  while(lengthSoFar < length) {
+    let token = getLoremIpsumToken()+" ";
+    lengthSoFar += token.length;
+    result.push(token)
+  }
+  
+  return result.join("").substr(0,length);
+}
+
+// Create a given number of paragraphs of a given length (in characters)
+// of lorem ipsum, formatted like:
+//   <p>Lorem ipsum dolor sit ame</p>
+// This is strictly for making big/slow posts for test purposes.
+function makeLoremIpsumBody(numParagraphs, paragraphLength) {
+  let result = [];
+  
+  for(var ii=0; ii<numParagraphs; ii++) {
```

Comment:
Why????

Reply:
Actual reason: I read a blog post arguing for `ii` rather than `i` a long time ago, adopted it, and forgot why.

I tracked down what that blog post would have said, and the only argument I see people making is that `ii` is easier to search for. But that doesn't apply when your search mechanism handles token boundaries correctly, so I don't have any reason other than habit now.